spec/models/miq_queue_spec.rb

```diff
@@ -986,4 +986,60 @@ def queue_items
       )
     end
   end
+
+  describe ".messaging_client_options" do
+    before do
+      config_file = Rails.root.join("config", "messaging.yml")
+      FileUtils.cp(Rails.root.join("config", "messaging.kafka.yml"), config_file) unless config_file.file?
```

Comment:
What cleans up from this?

Reply:
Also note that messing with the filesystem can break parallel tests, so i'm not sure this is a good idea. (can't think of how to do it otherwise without some sort of tempfile.

Reply:
I switched to stubbing `YAML.load_file`